### PR TITLE
labsite: Fix reloading for labs with edits

### PR DESCRIPTION
**Pull request by juliaogris**
When you edit a lab and reload the page you land on #welcome where the
editor is hidden. This means it appears on reload editor contents are
lost on the labsite.

Fix it in this PR.

In a preparatory commit rework the YouTube embed to be a screenshot link to
avoid Chrome dev console warnings and third party cookies.

In a follow-up commit we ensure to have a slightly more meaningful title
for code loaded via `#content` or `#source` hash

---

Requires a bit of manual testing, please.
https://evy-lang-stage-lab--432-2gxeqxg9.web.app/#content=H4sIAAAAAAAAA1XIMQqAMAxG4T2n+GcFSRyLPYy0AQNthYLa3l7UyeXB98yVIxNdmyWFYYEwMxMANDiPupYoL/ufeT8VbRBm9Kf03mA1JAVP82eDh2GEkJZINwcAGvJtAAAA

### CODE REVIEW

File: frontend/lab/samples/intro/welcome.htmlf [hunk 1]
```diff
@@ -7,12 +7,6 @@
...
-  src="https://www.youtube-nocookie.com/embed/FBqyiU4iZNY"
-  title="YouTube video player for Introductory Lab Walk Through"
-  frameborder="0"
-  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-  referrerpolicy="strict-origin-when-cross-origin"
-  allowfullscreen
-  class="youtube"
-></iframe>
+<a href="https://youtu.be/FBqyiU4iZNY" target="_blank" class="youtube">
+  <img src="samples/intro/img/youtube.jpeg" alt="YouTube screenshot of walk through" />
```
Comment (camh-) [inline frontend/lab/samples/intro/welcome.htmlf:11]: nit: the alt text should perhaps say "Screenshot of YouTube walk through". Same in the .html file

**Review by camh- — COMMENTED** [review]
The commit message for the second commit says "infect" where it should say "in fact". I fixed this in the PR description back when there was one commit and that would have been the squash merge commit so the message on the commit itself didn't matter. But now you've got two commits, you'll 1) need to fix the commit message and 2) should probably shorten the PR description as the detail will now be on the individual commit.

File: frontend/lab/samples/intro/welcome.htmlf (hunk 1 above)
Comment (juliaogris) [reply frontend/lab/samples/intro/welcome.htmlf]: thanks, done.

**Review by camh- — APPROVED** [review]
:green_apple:

**Pull request merged**